index.html

```diff
@@ -0,0 +1,526 @@
+          </div>
+        </section>
+      </div>
+    </div>
+  <!--2) BLUE BACKGROUND END-->
+
+
+  <!--3) GRADIENT BACKGROUND START-->
+    <div class="bg-gradient">
+      <!--ABOUT FLO SECTION-->
+      <section class="conversation-card content-wrap -cup">
+        <div class="about-flo-icon__bg"></div>
+        <div class="about-flo-icon"></div>
+        <div class="narrow-padding about-flo">
+          <div class="-content">
+            <p class="text -medium -color-lightblue -caps -bold centered">She gets to know you</p>
+            <p class="text -small -serif">Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+            <p class="text -small -serif">This conversation, which unfolds over time, feels less like twenty questions and more like the start of a relationship.</p>
+          </div>
+          <div class="-chat">
+            <div class="bubble bot-avatar flo" data-delay="0">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>Hi! I'm Florence. What's your name?</p>
+            </div>
+            <div class="bubble -response" data-delay="2000">
+              <p>Mike</p>
+            </div>
+            <div class="bubble flo" data-delay="2900">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>Nice to meet you, Mike. I can help you set up your account with a few simple questions.</p>
+            </div>
+            <div class="bubble flo" data-delay="4900">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>You can answer these now or whenever you like. I'm always here!</p>
+            </div>
+            <div class="bubble bot-avatar flo" data-delay="6700">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>First question: how old are you?</p>
+            </div>
+            <div class="bubble -response" data-delay="8700">
+              <p>65</p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+
+      <section class="conversation-card content-wrap -pills">
+        <div class="about-flo-icon__bg"></div>
+        <div class="about-flo-icon"></div>
+        <div class="narrow-padding about-flo">
+          <div class="-content">
+            <p class="text -medium -color-lightblue -caps -bold centered">SHE KEEPS TRACK OF YOUR REGIMEN.</p>
```

Comment:
Each of these sections should have a `h` tag for it's header to create the right structure.